docs/en/transform/nulltf.md

```diff
@@ -0,0 +1,67 @@
+# Nulltf
+
+## Description
+
+set default value for null field
+
+:::tip
+
+This transform only supported by engine Spark.
+
+:::
+
+## Options
+
+| name                | type    | required | default value |
+| ------------------- | ------- | -------- | ------------- |
+| fields              | array   | no       | -             |
+
+### fields [list]
+
+A list of fields whose default value will be set. 
+The default value of the field can be set in the form of "=value" after the field name. 
+If there is no "=value" after the field name, the default value will be set according to the field type.
+
+## Examples
+
+the configuration
+
+```bash
+  nulltf {
+      fields = ["name","price=0","num=100","flag","dt_timestamp=2022-05-18 13:51:40.603","dt_date=2022-05-19"]
+  }
```

Comment:
```suggestion
      fields : {
          price:0
          }
  }
```
Can you use the conf style here, then you don't need to parse "=".

Reply:
Yes, I will resubmit later.
Thanks for your suggestion :)